eth-custodian/contracts/test/NearProverMock.sol

```diff
@@ -1,31 +1,31 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
-pragma solidity ^0.6.12;
+pragma solidity ^0.8;
 
-import 'rainbow-bridge/contracts/eth/nearprover/contracts/INearProver.sol';
+import 'rainbow-bridge-sol/nearprover/contracts/INearProver.sol';
 
 contract NearProverMock is INearProver {
     function proveOutcome(
-        bytes memory proofData, 
-        uint64 blockHeight
+        bytes memory /*proofData*/,
+        uint64 /*blockHeight*/
```

Comment:
this is a file intended just for the test and names of variables there are for some clariy. Natspec won't be the best option as it would anyway require naming those params